test/unit/webdriver/device/keyboard_test.py

```diff
@@ -32,3 +32,69 @@ def test_hide_keyboard(self):
             appium_command('/session/1234567890/appium/device/hide_keyboard')
         )
         assert isinstance(driver.hide_keyboard(), WebDriver)
+
+    @httpretty.activate
+    def test_press_keycode(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/press_keycode'),
+            body='{"value": "86"}'
```

Comment:
This line is unnecessary, so delete this line.
This body simulates the value which appium server returns.

Reply:
@KazuCocoa please have a check again.

As tips, you will pass tests when both L42 is deleted and L42 is `body={}`.

Reply:
@ki4070ma I don't understand why don't we need to mock the response? I would agree on @KazuCocoa @manoj9788 way of writing it.

Reply:
Hum, in my opinion, its value (`body={blabla}`) doesn't impact to any other codes and result.

Reply:
@ki4070ma Feel free to approve it. I think this is the right way to mock it.

Reply:
Either is fine to add a response or not as a client-side check. (It depends on the case if the client would like to check internal parsing against the response.)
btw, the return value by the server is like `{ value: '' }`, i remember.

Reply:
Thanks, guys! so many confusions on this. 
I think mocking and validating the request is always the way to go and doesn't make sense to rely on the server having the values and it wouldn't impact if there are any bug from the server-side since this is a completely client-side unit tests.

Reply:
I'm merging this @KazuCocoa.

Reply:
ok, thanks for many your comments.

[JFYI]
> btw, the return value by the server is like { value: '' }, i remember.
Seems right.

> [debug] [WD Proxy] Got response with status 200: {"sessionId":"80c32258-b7fa-4f1e-bc5b-b76b11d9a082","value":null}

```
[HTTP] --> POST /wd/hub/session/b7a19413-0f4f-4a52-ab52-b9aa630cffb8/appium/device/press_keycode
[HTTP] {"keycode":86}
[debug] [W3C (b7a19413)] Calling AppiumDriver.pressKeyCode() with args: [86,null,null,null,null,"b7a19413-0f4f-4a52-ab52-b9aa630cffb8"]
[debug] [WD Proxy] Matched '/appium/device/press_keycode' to command name 'pressKeyCode'
[debug] [WD Proxy] Proxying [POST /appium/device/press_keycode] to [POST http://localhost:8200/wd/hub/session/80c32258-b7fa-4f1e-bc5b-b76b11d9a082/appium/device/press_keycode] with body: {"keycode":86,"metastate":null,"flags":null}
[debug] [WD Proxy] Got response with status 200: {"sessionId":"80c32258-b7fa-4f1e-bc5b-b76b11d9a082","value":null}
[debug] [W3C (b7a19413)] Responding to client with driver.pressKeyCode() result: null
[HTTP] <-- POST /wd/hub/session/b7a19413-0f4f-4a52-ab52-b9aa630cffb8/appium/device/press_keycode 200 28 ms - 14
```